rally_ovs/plugins/ovs/ovsclients_impl.py

```diff
@@ -289,6 +289,82 @@ def db_set(self, table, record, *col_values):
             self.run("set", args=args)
 
     def create_client(self):
-        print "*********   call OvnNbctl.create_client"
```

Comment:
Unrelated change. Please submit a new patch.

Reply:
ok I will address that in separate patch